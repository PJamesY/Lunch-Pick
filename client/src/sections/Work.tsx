import React from 'react';
import { Link } from 'react-router-dom';
import { StyledSection } from '../styledComponent/Section';

const Work = function Work() {
  return (
    <StyledSection id="work">
      <Link to="/pokemon">pokemon</Link>
    </StyledSection>
  );
};

export default Work;
