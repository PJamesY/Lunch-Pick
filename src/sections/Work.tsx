import React from 'react';
import { StyledSection } from '../styledComponent/Section';

const Work = function Work() {
  return <StyledSection id="work">work</StyledSection>;
};

export default Work;
