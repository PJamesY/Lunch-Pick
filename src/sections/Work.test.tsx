import React from 'react';
import { shallow } from 'enzyme';
import Work from './Work';

describe('Work section', () => {
  it('should match work snap shot', () => {
    expect(shallow(<Work />)).toMatchSnapshot();
  });
});
