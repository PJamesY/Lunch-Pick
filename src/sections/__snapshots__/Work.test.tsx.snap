// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Work section should match work snap shot 1`] = `ShallowWrapper {}`;
